src/Integration.Vsix.UnitTests/CFamily/CFamilyEmbeddedSonarWayRulesTests.cs

```diff
@@ -44,9 +44,8 @@ public class CFamilyEmbeddedSonarWayRulesTests
         // The QP filter has "active/inactive" tabs. The number of rules is shown in the top-right of the screen.
         // 5. Repeat for C++.
 
-        // Rule data for C-Family plugin v6.47.0.62356
+        // Rule data for C-Family plugin v6.48.0.62520
```

Comment:
Looking at the release notes, there don't appear to be any changes in the number of rules or the contents of Sonar Way.